metaflow/plugins/aws/step_functions/step_functions.py

```diff
@@ -813,7 +813,8 @@ def batch(self, job):
             .parameter('RetryStrategy', 
                 to_pascalcase(job.payload['retryStrategy'])) \
             .parameter('Timeout', 
-                to_pascalcase(job.payload['timeout']))
+                to_pascalcase(job.payload['timeout'])) \
+            .parameter('Tags', job.payload['tags'])
```

Comment:
might need to add `propagateTags` somewhere here as well.

Reply:
The `job-definition` will take care of setting `propagateTags` to `True` always, so setting it here is un-neccessary.